velox/connectors/hive/storage_adapters/hdfs/HdfsWriteFile.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <hdfs/hdfs.h>
+#include "velox/common/file/File.h"
+
+namespace facebook::velox {
+
+/// Implementation of hdfs write file. Nothing written to the file should be
+/// read back until it is closed.
+class HdfsWriteFile : public WriteFile {
+ public:
+  /// The constructor.
+  /// @param hdfs The configured hdfs filesystem handle.
+  /// @param path The file path to write.
+  HdfsWriteFile(hdfsFS hdfs, std::string_view path);
+
+  /// The constructor.
+  /// @param hdfs The configured hdfs filesystem handle.
+  /// @param path The file path to write.
+  /// @param bufferSize Size of buffer for write - pass 0 if you want
+  /// to use the default configured values.
+  /// @param replication Block replication - pass 0 if you want to use
+  /// the default configured values.
+  /// @param blockSize Size of block - pass 0 if you want to use the
+  /// default configured values.
+  HdfsWriteFile(
```

Comment:
`hdfs` -> `hdfsClient`
remove the above constructor.
Add default `0` to this one.